doc/theory/glossary.tex

```diff
@@ -168,10 +168,11 @@
 	\begin{equation*}
 	w : \langle G \rangle \to \mathbb R \cup \{\infty\}
 	\end{equation*}
-	given by
-	\begin{equation*}
-	w(\sigma) = \max(\{ w(e)\ |\ e \in G \cap \sigma\})
-	\end{equation*}
+	defined respectively by 
+	\begin{align*}
+	w\{v_0, \dots, v_n\} & = \max\{ w\{v_i, v_j\}\ |\ i < j\} \\
```

Comment:
Great! Though i < j is not necessary in the first case. Maybe I'd have enclosed the sets in parentheses but this is just personal style.

Reply:
Though one would then have to also add redundant parentheses around the tuples below, which is basically correct but unacceptable in real life.

Reply:
The i < j in the first line could be replaces with i \neq j. I am fine with either. About enclosing, that was my thought also.

Reply:
Thanks for the reply. I meant to say that, since you have sets, you don't have to restrict the indices. That is, $\max\{ w\{v_i, v_j\}\ | \}_{i, j = 0}^n = \max\{ w\{v_i, v_j\}\ |\ i < j\}$. Again though, it's not like the current version is wrong. Feel free to make a decision and (squash and) merge!

Reply:
The left hand side computes w(v_i, v_i) which is not defined, though. I Squash and merge :-)